include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -2,7 +2,6 @@
 
 #include <algorithm> // transform
 #include <array> // array
-#include <ciso646> // and, not
```

Comment:
This is too easy. Older versions of MSVC still need this header (see https://ci.appveyor.com/project/nlohmann/json/builds/27972282). We should use Hedley macros to detect C++20 compilers and only exclude the header if we are sure to work on C++20.